portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -13,20 +14,36 @@
 // limitations under the License.
 
 package com.google.sps.servlets;
-
+import java.util.ArrayList;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.gson.*;
```

Comment:
Don't import whole library. This will import all files in this package. Only imports the file needed.